cscs-checks/apps/quantumespresso/quantumespresso_check.py

```diff
@@ -134,53 +173,31 @@ def set_cpu_binding(self):
 
 @rfm.simple_test
 class QuantumESPRESSOGpuCheck(QuantumESPRESSOCheck):
-    def __init__(self):
-        super().__init__()
+    valid_systems = ['daint:gpu']
+    num_gpus_per_node = 1
+    num_tasks_per_node = 1
+    num_cpus_per_task = 12
+
+    @run_after('init')
+    def set_reference(self):
+        if self.scale == 'small':
+            self.reference = REFERENCE_GPU_PERFORMANCE_SMALL
+            self.energy_value = -11427.09017168
```

Comment:
These references should be defined in a base class for the GPU. @hurricane642, could please add a `#FIXME: check the tolerances and the reference difference between the CPU and GPU tests?` 

Side note: @jjotero We will need to review these numbers later. They seem a bit strange. Look at the tolerances. They are one order of magnitude tighter than the CPU ones. This is strange.